src/ebfloeseg/load.py

```diff
@@ -0,0 +1,103 @@
+from enum import Enum
+import logging
+from pathlib import Path
+from typing import Annotated
+import requests
+
+import typer
+
+_logger = logging.getLogger(__name__)
+
+
+class ImageType(str, Enum):
+    truecolor = "truecolor"
+    cloud = "cloud"
+    landmask = "landmask"
+
+
+def get_width_height(bbox: str, scale: float):
+    """Get width and height for a bounding box where one pixel corresponds to `scale` bounding box units
+
+    Examples:
+        >>> get_width_height("0,0,1,1", 10)
+        (10, 10)
+
+        >>> get_width_height("0,0,1,5", 10)
+        (2, 10)
+
+        >>> get_width_height("0,0,1,5", 10)
+        (2, 10)
+
+    """
+    x1, y1, x2, y2 = [float(n) for n in bbox.split(",")]
+    x_length = abs(x2 - x1)
+    y_length = abs(y2 - y1)
+
+    width, height = int(x_length / scale), int(y_length / scale)
+    return width, height
+
+
+app = typer.Typer()
+
+
+@app.command()
+def main(
+    outfile: Annotated[Path, typer.Argument()],
+    datetime: str = "2016-07-01T00:00:00Z",
+    wrap: str = "day",
+    kind: ImageType = ImageType.truecolor,
+    bbox: str = "-2334051.0214676396,-414387.78951688844,-1127689.8419350237,757861.8364224486",
```

Comment:
How do you mean? Like `namedtuple("BBox", "x1,y1,x2,y2")`? 
I think that might make a lot of sense. 
I don't want to do that right now because this bit gets updated later to accept strings in a few different input formats [specifically `x1,y1,x2,y2` and `(x1, y1, x2, y2)`] later, so it can support cylc.

Reply:
Did I misunderstand – did you mean something like this? 
```python
@dataclass
class DataSet:
    datetime: str
    wrap: str
    satellite: Satellite
    kind: ImageType
    bbox: BoundingBox
    scale: int
    crs: str
    ts: int


ExampleDataSetBeaufortSea = DataSet(
    datetime="2016-07-01T00:00:00Z",
    wrap="day",
    satellite=Satellite.terra,
    kind=ImageType.truecolor,
    scale=250,
    bbox=(
        -2334051,
        -414387,
        -1127689,
        757861,
    ),
    crs="EPSG:3413",
    ts=1683675557694,
)


def load(
    datetime: str = ExampleDataSetBeaufortSea.datetime,
    wrap: str = ExampleDataSetBeaufortSea.wrap,
    satellite: Satellite = ExampleDataSetBeaufortSea.satellite,
    kind: ImageType = ExampleDataSetBeaufortSea.kind,
    bbox: BoundingBox = ExampleDataSetBeaufortSea.bbox,
    scale: int = ExampleDataSetBeaufortSea.scale,
    crs: str = ExampleDataSetBeaufortSea.crs,
    ts: int = ExampleDataSetBeaufortSea.ts,
    format: str = "image/tiff",
    validate: bool = True,
) -> LoadResult:
```

In any case, your comment gave me the idea, and I've done that in #48

Reply:
Right, something very much like that!